### PR TITLE
Docker file added to formsflow-web 

### PR DESCRIPTION
**Pull request by shuhaib-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-2365
Issue Type: BUG/ FEATURE

# Changes

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: forms-flow-web-root-config/sample.env [hunk 1]
```diff
@@ -21,10 +21,10 @@
 #MF_FORMSFLOW_THEME_URL=https://s3.ap-northeast-1.amazonaws.com/formsflow.ai-micro-front-ends/forms-flow-theme/forms-flow-theme.gz.js
 
 #TODO only for Customised Local delevelopment with code from https://github.com/AOT-Technologies/forms-flow-ai-micro-front-ends  use below default values instead
-#MF_FORMSFLOW_NAV_URL=//{your-ip-address}:8080/forms-flow-nav.js
-#MF_FORMSFLOW_ADMIN_URL=//{your-ip-address}:8001/forms-flow-admin.js
-#MMF_FORMSFLOW_SERVICE_URL=://{your-ip-address}:8081/forms-flow-service.js
-#MF_FORMSFLOW_THEME_URL=//{your-ip-address}:8082/forms-flow-theme.js
+#MF_FORMSFLOW_NAV_URL=//{your-ip-address}:3005/forms-flow-nav.js
```
Comment (abhilash-aot) [inline forms-flow-web-root-config/sample.env:24]: Need to modify the ports in the sample.env local reference

**Review by abhilash-aot — APPROVED** [review]

File: forms-flow-web-root-config/sample.env (hunk 1 above)
Comment (shuhaib-aot) [reply forms-flow-web-root-config/sample.env:24]: Yes this is sample.env

**Review by abilpraju-aot — APPROVED** [review]

File: forms-flow-web/src/components/PrivateRoute.jsx [hunk 2]
```diff
@@ -141,7 +141,7 @@ const PrivateRoute = React.memo((props) => {
               userRoles.includes(STAFF_DESIGNER) ? (
                 <Component {...props} />
               ) : (
-                <>unauthorized</>
+                <Redirect exact to="/404" />
```
Comment (sumesh-aot) [inline forms-flow-web/src/components/PrivateRoute.jsx:144]: Why is Unauthorized going to 404 ? We should be having Not Found and Unauthorized separate, right ?

File: forms-flow-web-root-config/sample.env (hunk 1 above)
Comment (abhilash-aot) [reply forms-flow-web-root-config/sample.env:24]: ok Got it

**Review by abhilash-aot — APPROVED** [review]

**Pull request merged**